### ISSUE TITLE
Wrong inputs in `gvar.BufferDict.uniform` can break permanently `BufferDict.get`

### ISSUE THREAD

**Issue by Gattocrucco** [gplepage/gvar#30]
```python
import gvar
gvar.BufferDict.uniform(0, 1, 2) # pass an integer instead of a label
b = gvar.BufferDict() # an unrelated empty bufferdict
b.get('cippa', 3.14) # try to get a missing key with backup value -> error instead of 3.14
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/_collections_abc.py", line 819, in get
    return self[key]
  File "_bufferdict.pyx", line 321, in gvar._bufferdict.BufferDict.__getitem__
TypeError: unsupported operand type(s) for +: 'int' and 'str'
```

This is annoying when I make a mistake calling `gvar.BufferDict.uniform` and then have to restart the ipython session.

**Comment by gplepage** [comment]
This is trivial to fix, of course. I will post an update later today or tomorrow.

**Comment by gplepage** [comment]
Fixed in version 11.10.1.

**Comment by Gattocrucco** [comment]
thanks
